From c0f24d76a7e9c7156ffbddebe97a88ef39c5525e Mon Sep 17 00:00:00 2001
From: Jakub Jurek <jakubjurek945@gmail.com>
Date: Sat, 2 Mar 2024 17:38:41 +0100
Subject: [PATCH] fix prinfs

---
 ether-wake.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ether-wake.c b/ether-wake.c
index eef22a7..84454d5 100644
--- a/ether-wake.c
+++ b/ether-wake.c
@@ -131,7 +131,7 @@ int main(int argc, char *argv[])
 		case 'D': debug++;			break;
 		case 'i': ifname = optarg;	break;
 		case 'p': get_wol_pw(optarg); break;
-		case 'u': printf(usage_msg); return 0;
+		case 'u': printf("%s",usage_msg); return 0;
 		case 'v': verbose++;		break;
 		case 'V': do_version++;		break;
 		case '?':
@@ -140,7 +140,7 @@ int main(int argc, char *argv[])
 	if (verbose || do_version)
 		printf("%s\n", version_msg);
 	if (errflag) {
-		fprintf(stderr, brief_usage_msg);
+		fprintf(stderr, "%s", brief_usage_msg);
 		return 3;
 	}
 
